core/src/com/mygdx/game/steerers/FollowPathSteerer.java

```diff
@@ -36,6 +43,45 @@
  */
 public class FollowPathSteerer implements Steerer {
 
+	// This should become part of gdx-ai since the ability to know
+	// the selected behavior is generally useful.
+	private static class MyPrioritySteering extends PrioritySteering<Vector3> {
+		protected int selectedBehaviorIndex;
+
+		public MyPrioritySteering (Steerable<Vector3> owner, float epsilon) {
+			super(owner, epsilon);
+		}
+		
+		public int getSelectedBehaviorIndex() {
```

Comment:
It sounds appropriate, but I am not sure if I am familiar enough with the steering API (aside from path following) to give an informed opinion unfortunately.